src/Tribe/JSON_LD/Organizer.php

```diff
@@ -59,4 +59,21 @@ public function get_data( $post = null, $args = array( 'context' => false ) ) {
 		return array( $post_id => $data );
 	}
 
+	/**
+	 * Get a link to the event
+	 *
+	 * @param  int|WP_Post  $post The Post Object or ID
+	 *
+	 * @return false|string Link to the event or false
+	 */
+	protected function get_link( $post ) {
+		if ( class_exists( 'Tribe__Events__Pro__Main' ) ) {
```

Comment:
I'm not a huge fan of the half-step but for now I suppose it'll have to suffice in order to get this fix out there. Adding it to that overhaul would work. Could you also throw a `@TODO` on this and the venue if statement that points at the overhaul ticket and advertises that we don't like this hacky solution and it should be fixed at a later date?